activities/domain.v1/domain.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.domain.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.domain.v1";
+option go_package = "activities.v1";
+
+import "google/protobuf/timestamp.proto";
+import "logistics/domain/v1/domain.proto";
+import "global/domain/v1/domain.proto";
+
+
+message Activity {
+    // reference the correlating activities
+    string correlation_id = 1;
+    string entity_uuid = 2;
+    google.protobuf.Timestamp timestamp = 3;
+    ActivityType.Type type = 4;
+    // subject who causes the activity
+    SubjectReference subject_causing = 5;
+
+    // reference the target the activity applies to
+    Reference target = 6;
+
+    // additional named references to subjects or resources
+    map<string, Reference> references = 7;
+
+}
+
+message Reference {
+    oneof reference {
+        ResourceReference resource_ref = 1;
+        SubjectReference subject_ref = 2;
+    }
+}
+
+message ResourceReference {
+    string id = 1;
+    io.spoud.sdm.global.domain.v1.ResourceEntity.Type entity_type = 2;
+}
+
+message SubjectReference {
+    string id = 1;
+}
+
+message ActivityType {
+    enum Type {
+        UNKNOWN = 0;
+        // 100..199 related to ownership change
+        OWNER_CHANGE = 100;
```

Comment:
since we are building the msg server side probably we will not need that. what you think @gaetancollaud @spike83? I didnt remove it because we may want to do it different in a near future.

Reply:
remove everything that's not needed. It's easier to add stuff than maintaining stuff.